### DISCUSSION TITLE
Is there anyway to create component with additional methods?

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#665] [Q&A]
Hello Sketchware team,
Your app is helping very much to develop my own project but I am facing a problem now.
I can create components but I can't add additional method in it.

Like
![IMG_20221201_182703](https://user-images.githubusercontent.com/96680248/205059169-ecc80d1d-5ff7-44be-a969-8c9d94fd8603.jpg)

Is there anyway to create like above methods with component.

A lot of thanks to Sketchware team specially @JavkhlanK to help me a lot.

**Comment by DevVigilante** [comment]
Why there is no discussion for this topic??😓😓😥😢

**Reply by Majestychen** [reply]
This feature may not be provided.
